### PR TITLE
ZCS-1057: Universal UI - user should be able to maximize & minimize the columns smoothly

### PR DESCRIPTION
**Pull request by kalpeshhpatel**
Changes:

* ZmZimbraMail:
  * Change threshold of sash from `20` to `5` to have smooth effect and also to be consistent with other threshold
* skin.properties:
  * Added/updated required properties

https://jira.corp.synacor.com/browse/ZCS-1057

### CODE REVIEW

File: WebRoot/skins/_base/base4/skin.properties [hunk 1]
```diff
@@ -987,26 +987,26 @@ PropertySheetNoLeftSpacing  = padding-left:0px;
 ################
 #   SASH - splits a pane
 ################
-SashSize                    = 5px
-SashPadding                 = 1px
-Sash                        =
-SashContainer               = height:100%;
+SashSize                    = 1px
+SashPadding                 = 0px
```
Comment (SatyamKumarOnline) [inline WebRoot/skins/_base/base4/skin.properties]: I believe we donot name the property based on css property.

Comment (kalpeshhpatel) [inline WebRoot/skins/_base/base4/skin.properties]: Those properties are already defined, I didn't created it though. And it just declares or set absolute value and not the css property, so it sounds okay to me. Let me know if you have any question.

File: WebRoot/skins/_base/base4/skin.properties [hunk 2]
```diff
@@ -987,26 +987,26 @@ PropertySheetNoLeftSpacing  = padding-left:0px;
...
 
 ################
 #   HORIZONTAL SASH - splits a pane into two tall panels
 ################
 HSashSize                   = height:100%; padding:0px @SashPadding@; width:@SashSize@;
-HSash                       = @HSashSize@
-HSashContainer              = @ColResizeCursor@
+HSash                       = @HSashSize@ @Sash@
+HSashContainer              = @ColResizeCursor@; width:10px;
 HSashThumb                  = margin-left:-1px; margin-top:-7px;
```
Comment (silentsakky) [inline WebRoot/skins/_base/base4/skin.properties]: If we are not showing sash thumb image then please remove commented code from
templates/dwt/Widgets.template
dwt.Widgets#DwtVerticalSash
dwt.Widgets#DwtHorizontalSash

and also remove images from img directory, if not used anywhere

this is not important from functionality point of view but will have to have clean code for better understanding

File: WebRoot/skins/_base/base4/skin.properties (hunk 1 above)
Comment (SatyamKumarOnline) [inline WebRoot/skins/_base/base4/skin.properties]: ok. Got that.

File: WebRoot/skins/_base/base4/skin.properties (hunk 2 above)
Comment (kalpeshhpatel) [inline WebRoot/skins/_base/base4/skin.properties]: Fixed: https://github.com/Zimbra/zm-web-client/pull/24/files

**Pull request merged**